### PR TITLE
Make PXE menu functional, remove 32-bit entries

### PR DESCRIPTION
**Pull request by Lekensteyn**
While testing the automated iso I noticed that the included kernel was not the LTS one as advertised on the website. Apparently the PXE boot method was not tested, so here it is. [By the way, the LTS kernel does not seem to be included at all, there is an initramfs-linux.img, but no initramfs-linux-lts.img),

PXE supports three methods to serve the rootfs, so submenus were created for each of them (scripted, to reduce human error).

Booting over PXE has the advantage that there is no need to burn a CD or write to a slow USB stick. Tested with PXE and HTTP following instructions on https://wiki.archlinux.org/index.php/PXE

### CODE REVIEW

File: iso/syslinux/archiso_pxe64.cfg [hunk 1]
```diff
@@ -1,32 +1,14 @@
...
-Boot the Arch Linux (x86_64) live medium (Using NBD).
-It allows you to install Arch Linux or perform system maintenance.
-ENDTEXT
-MENU LABEL Boot Arch Linux (x86_64) (NBD)
-LINUX boot/x86_64/vmlinuz
-INITRD boot/x86_64/initramfs-linux.img
-APPEND archisobasedir=%INSTALL_DIR% archisolabel=%ARCHISO_LABEL% archiso_nbd_srv=${pxeserver}
-SYSAPPEND 3
+MENU BEGIN arch64_nbd
+MENU LABEL Boot varous Arch Linux (x86_64) live media using NBD
```
Comment (pierremoreau) [inline iso/syslinux/archiso_pxe64.cfg]: Small typo to "various" :-)

**Comment by pierremoreau** [comment]
Thanks for the pull request! I never used PXE, so I never tested. I don’t know whether @hakzsam did.
LGTM. I’ll try to test it on Wednesday evening. Ping again, if you have no news by the week-end.

Regarding the LTS kernel, you seem absolutely right. @hakzsam any clue? I’ll investigate that.

**Comment by hakzsam** [comment]
No ideas, I didn't test the iso recently but I think you are right as well.

File: iso/syslinux/archiso_pxe64.cfg (hunk 1 above)
Comment (Lekensteyn) [inline iso/syslinux/archiso_pxe64.cfg]: Fixed, obviously not everything was generated in an automated way :-P

**Comment by Lekensteyn** [comment]
PXE is really cool, it makes testing a kernel + initrd remotely (~a few meter) on a diskless system really easy (which is why I used it in the past). (For some reason the lpxelinux.0 file did not work for me though so I had to add a `pxelinux.0` file.)

**Comment by Lekensteyn** [comment]
@pierremoreau Friendly ping :-)

Would it also be possible to include the repo version on https://nouveau.pmoreau.org/ so it is known how the image is generated?

**Comment by pierremoreau** [comment]
I started to build an image with your fixes, so I’ll be able to test it tomorrow. I’ll add a commit reference of archlive-nouveau to each new entry, good idea.
And, that made me realise I have some changes to upstream. :-)

**Comment by pierremoreau** [comment]
Got an issue with the compile script which made the image compilation failed. Working on fixing that.

**Comment by pierremoreau** [comment]
Ok, I tested the HTTP version and it worked fine! I got some errors thought:
```
Jul 26 22:54:28 Thessia dnsmasq-tftp[1258]: error 0 TFTP Aborted received from 192.168.0.141
Jul 26 22:54:28 Thessia dnsmasq-tftp[1258]: failed sending /mnt/archiso/arch/boot/syslinux/lpxelinux.0 to 192.168.0.141
Jul 26 22:54:28 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/lpxelinux.0 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch//boot/isolinux/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch//isolinux/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch//boot/syslinuxldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: error 0 No error, file close received from 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: failed sending /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/whichsys.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/libcom32.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/libcom32.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: error 0 No error, file close received from 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: failed sending /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe_choose.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/ifcpu64.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/libcom32.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/libcom32.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: error 0 No error, file close received from 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: failed sending /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/ldlinux.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/ldlinux.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe_both_inc.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_head.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_nbd.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_nfs.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_http.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_tail.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/vesamenu.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/libcom32.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/libcom32.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: file /mnt/archiso/arch/libutil.c32 not found
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch//boot/syslinux/libutil.c32 to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe_both_inc.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_head.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_nbd.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_nfs.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_pxe64_http.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/archiso_tail.cfg to 192.168.0.141
Jul 26 22:54:29 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/syslinux/splash.png to 192.168.0.141
Jul 26 22:54:52 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/x86_64/vmlinuz-linux-latest to 192.168.0.141
Jul 26 22:54:55 Thessia dnsmasq-tftp[1258]: sent /mnt/archiso/arch/boot/x86_64/initramfs-linux-nouveau.img to 192.168.0.141
```
Maybe the `build.sh` script needs some updates? Or it is something expected, dunno.

**Pull request merged**